src/test/ui/async-await/issue-64130-non-send-future-diags.stderr

```diff
@@ -1,14 +1,14 @@
-error[E0277]: `std::sync::MutexGuard<'_, u32>` cannot be sent between threads safely
-  --> $DIR/issue-64130-non-send-future-diags.rs:23:5
+error: future cannot be sent between threads safely
+  --> $DIR/issue-64130-non-send-future-diags.rs:21:5
    |
-LL | fn is_send<T: Send>(t: T) {
+LL | fn is_send<T: Send>(t: T) { }
    |    -------    ---- required by this bound in `is_send`
 ...
 LL |     is_send(foo());
-   |     ^^^^^^^ `std::sync::MutexGuard<'_, u32>` cannot be sent between threads safely
+   |     ^^^^^^^ future returned by `foo` is not send
    |
    = help: within `impl std::future::Future`, the trait `std::marker::Send` is not implemented for `std::sync::MutexGuard<'_, u32>`
-note: future does not implement `std::marker::Send` as this value is used across an await
+note: future is not send as this value is used across an await
```

Comment:
I think this should be capitalized and in backticks (i.e. 'future is not `Send`')

Reply:
You're right, [nagisa mentioned that already](https://rust-lang.zulipchat.com/#narrow/stream/187312-wg-async-foundations/topic/future-not-send.20diagnostic.20.2364130/near/177478941) and I forgot.

Reply:
Fixed!